h/static/scripts/tests/controllers/dropdown-menu-controller-test.js

```diff
@@ -29,21 +29,32 @@ describe('DropdownMenuController', () => {
     return menuEl.classList.contains('is-open');
   }
 
+  function clickMenuLink() {
+    toggleEl.dispatchEvent(new Event('click', { cancelable: true }));
```

Comment:
In most code we do. I think these tests were written before we started using syn.